### PR TITLE
1 proposal basic rendering

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Add support for CreateServiceNervousSystem proposal. Proposal style changes will come in a separate pr.

- [x] Related `ic-js` pr  https://github.com/dfinity/ic-js/pull/369
- [x] Upgrade `ic-js` in `nns-dapp` (done in [pr](https://github.com/dfinity/nns-dapp/pull/2805))

# Changes

- add CreateServiceNervousSystem labels
- extend `Json` component to render base_64 images.

# Tests

- `fromDefinedNullableRecursive`
- `Json` renders images

# Todos

- [x] Add entry to changelog (if necessary).

# Screenshots

image

## Payload
image

### CODE REVIEW

File: HOWTO.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ There are parts that usually change in a proposal:
 - A new proposal topic.
 - A new `nnsFunction` or changes in one.
 
-The change is not always just in one section. Many times you need to fix two sections simulataneously. For example, a new topic comes with a new `nnsFunction`.
```
Comment (mstrasinskis) [inline HOWTO.md:9]: Just a typo

File: frontend/src/lib/utils/utils.ts [hunk 2]
```diff
@@ -390,3 +390,21 @@ export const expandObject = (
     }
     return acc;
   }, {} as Record<string, unknown>);
+
+// Recursively apply fromDefinedNullable to all values in an array when the value the length of the array is 1
+export const fromDefinedNullableRecursive = (root: unknown): unknown => {
```
Comment (lmuntaner) [inline frontend/src/lib/utils/utils.ts:395]: We want to do that at the nns-js level.

We have todos there:

* https://github.com/dfinity/ic-js/blob/da291d796ae11af913470615bfda421548271e93/packages/nns/src/canisters/governance/response.converters.ts#L396
* https://github.com/dfinity/ic-js/blob/da291d796ae11af913470615bfda421548271e93/packages/nns/src/canisters/governance/request.converters.ts#L261

Comment (mstrasinskis) [reply frontend/src/lib/utils/utils.ts:395]: Oh, I didn't see the todos, makes sense. Will move the logic there, thanks.

**Comment by mstrasinskis** [comment]
@lmuntaner the response transformation was removed.

**Review by lmuntaner — APPROVED** [review]
LGTM! Thanks

You'll upgrade to new ic-js in another PR then?

**Pull request merged**